shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -140,7 +140,11 @@ export default class PartialClosure extends Component {
       customWalletUse: true,
       customWallet: this.wallets[buyToken.toUpperCase()],
       extendedControls: false,
+      estimateFeeValues: { /* ... */ },
```

Comment:
Думаю, лучше так назвать
```suggestion
      estimatedFeeValues: {},
```